docs/sources/flow/getting-started/migrating-from-prometheus.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Migrating from Prometheus
+weight: 320
+---
+
+# Migrating from Prometheus
+
+Migration to Grafana Agent Flow from [Prometheus][] can be done using built
+in Grafana Agent tooling.
+
+This topic describes how to:
+
+* Convert a Prometheus configuration to a Grafana Agent Flow configuration
+* Run a Prometheus configuration natively using Grafana Agent Flow
+
+[Prometheus]: https://prometheus.io/docs/prometheus/latest/configuration/configuration/
+
+## Components used in this topic
+
+* [prometheus.scrape][]
+* [prometheus.remote_write][]
+
+[prometheus.scrape]: {{< relref "../reference/components/prometheus.scrape.md" >}}
+[prometheus.remote_write]: {{< relref "../reference/components/prometheus.remote_write.md" >}}
+
+## Before you begin
+
+* Have an existing Prometheus configuration.
+* Have a set of Prometheus applications ready to push telemetry data to
+  Grafana Agent Flow.
+* Be familiar with the concept of [Components][] in Grafana Agent Flow.
+
+[Components]: {{< relref "../concepts/components.md" >}}
+[convert]: {{< relref "../reference/cli/convert.md" >}}
+[run]: {{< relref "../reference/cli/run.md" >}}
+[Flow Debugging]: {{< relref "../monitoring/debugging.md" >}}
+[debugging]: #debugging
+
+## Convert a Prometheus configuration
+
+In order to fully migrate your configuration from [Prometheus] to Grafana Agent
+Flow, your Prometheus configuration must be converted into a Grafana Agent Flow
+configuration. This will enable you to modify it going forward as a flow
+configuration and take full advantage of the many additional features available
+in Grafana Agent Flow.
+
+> In this task, we will use the [convert][] CLI command to output flow
+> configuration from a Prometheus configuration.
+
+1. Execute the following
```

Comment:
It feels like we leave the reader hanging after step 1. For example, let's say I just followed this guide. What do I do with OUTPUT_CONFIG_PATH now that I've run the command?

I think there should be a step 2 which instructs the user to make sure that they update their flow mode installation so it will load OUTPUT_CONFIG_PATH as the config file. Alternatively, we can instruct users to set OUTPUT_CONFIG_PATH to the default config location for their relevant system.

(In either case, we should link them to the relevant documentation pages rather than going into too much detail on how to do either of these things here)

Reply:
good point, step added